From ffe2b4445947bcb2aa9f6bb5e01367127dbf9252 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Thu, 3 Dec 2015 13:01:57 +0800
Subject: [PATCH 0762/5242] MLK-11947 ARM: dts: add missing dtb names in
 Makefile for imx7d-sdb

commit  c53d18d698bd5c12f964dda370b658b1a5592a4e from
https://source.codeaurora.org/external/imx/linux-imx.git

Making those dtb files to be complied by make dtbs command.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 0aa13d1..4a05a5f 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -597,6 +597,8 @@ dtb-$(CONFIG_SOC_IMX7D) += \
 	imx7d-pico-pi.dtb \
 	imx7d-sbc-imx7.dtb \
 	imx7d-sdb.dtb \
+	imx7d-sdb-wm8960.dtb \
+	imx7d-sdb-hdmi-audio.dtb \
 	imx7d-sdb-epdc.dtb \
 	imx7d-sdb-gpmi-weim.dtb \
 	imx7d-sdb-qspi.dtb \
-- 
1.7.9.5

